scripts/upgrade

```diff
@@ -42,15 +42,20 @@ ynh_clean_setup () {
 # Exit if an error occurs during the execution of the script
 ynh_abort_if_errors
 
+#=================================================
+# CHECK VERSION
+#=================================================
+
+upgrade_type=$(ynh_check_app_version_changed)
+
 #=================================================
 # ENSURE DOWNWARD COMPATIBILITY
 #=================================================
 ynh_script_progression --message="Ensuring downward compatibility..." --weight=1
 
-# If final_path doesn't exist, create it
-if [ -z "$final_path" ]; then
-	final_path=/var/www/$app
-	ynh_app_setting_set --app=$app --key=final_path --value=$final_path
+if [ -z "$admin" ]; then
+	admin="default_value"
```

Comment:
Ugh I'm afraid it's not gonna be that simple, because later this is supposed to be used as a user for the permissions, and Yunohost is gonna check that the user actually exists and may return an error code if it doesn't ...

Though I don't have any obvious simple way of finding out what user to use instead ...